### PR TITLE
Add script to mass-create pipelines

### PR DESCRIPTION
**Pull request by tkyi**
Adding script to mass-create pipelines

### CODE REVIEW

File: mass-create-pipelines.js [hunk 1]
```diff
@@ -0,0 +1,278 @@
...
+const config = require('./.secrets_config');
+
+const JWT = process.env.JWT || config.jwt;
+const GH_TOKEN = process.env.GH_TOKEN || config.ghToken; // *some Github tokens are named differently
+const NPM_TOKEN = process.env.NPM_TOKEN || config.npmToken;
+const K8S_TOKEN = process.env.K8S_TOKEN || config.k8sToken;
+const DOCKER_TRIGGER = process.env.DOCKER_TRIGGER || config.dockerTrigger;
+const COVERALLS_REPO_TOKEN = process.env.COVERALLS_REPO_TOKEN || config.coverallsRepoToken;
+const ACCESS_KEY = process.env.ACCESS_KEY || config.accessKey;
+const INSTANCE = process.env.ACCESS_KEY || config.instance || 'http://localhost:8080';
```
Comment (minz1027) [inline mass-create-pipelines.js]: typo?

File: README.md [hunk 2]
```diff
@@ -1,2 +1,10 @@
 # toolbox
 Central repository for handy Screwdriver-related scripts and other tools
+
+## To recreate all pipelines:
```
Comment (minz1027) [inline README.md]: Can we make it clear that we will also create all secrets except for the git_key

Comment (tkyi) [inline README.md]: added

File: mass-create-pipelines.js (hunk 1 above)
Comment (tkyi) [inline mass-create-pipelines.js]: thanks for the catch!

**Pull request merged**